Description: Fixes wrong requirements.txt
 The current Glance requirements breaks pbr, this patch fixes it.
Author: Thomas Goirand <zigo@debian.org>
Origin: upstream, https://review.openstack.org/#/c/166796/
Last-Update: 2015-03-23

--- glance-2015.1~b3.orig/requirements.txt
+++ glance-2015.1~b3/requirements.txt
@@ -52,7 +52,7 @@ oslo.db>=1.5.0  # Apache-2.0
 oslo.i18n>=1.3.0  # Apache-2.0
 oslo.log>=0.4.0 # Apache-2.0
 oslo.messaging>=1.6.0  # Apache-2.0
-oslo.policy~=0.3.1  # Apache-2.0
+oslo.policy>=0.3.1  # Apache-2.0
 oslo.serialization>=1.2.0               # Apache-2.0
 
 retrying>=1.2.3,!=1.3.0 # Apache-2.0
